viewlet/tests/urls.py

```diff
@@ -1,11 +1,20 @@
 # coding=utf-8
 from __future__ import unicode_literals
-try:
-    from django.conf.urls import patterns, include
-except ImportError:
-    from django.conf.urls.defaults import patterns, include
+import django
 
-urlpatterns = patterns(
-    '',
-    (r'^viewlet/', include('viewlet.urls')),
-)
+if django.VERSION[:2] < (1, 7):
+    try:
+        from django.conf.urls import patterns, include
+    except ImportError:
+        from django.conf.urls.defaults import patterns, include
```

Comment:
Also, I think it's not correct to use `except ImportError:` instead of version check. A lot of people do it but they are all wrong 😈